.github/ISSUE_TEMPLATE/component-cardano-cli.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report related to cardano-cli
+about: Report a bug related to cardano-cli
+title: "[BUG] - "
+labels: ["type: bug", "comp: cardano-cli"]
```

Comment:
We moved the cli and api out of cardano-node to avoid non-node related issues from being raised in cardano-node.